manifests/kubernetes/kubevirt-tekton-tasks-kubernetes.yaml

```diff
@@ -306,13 +308,15 @@ spec:
       default: ""
   steps:
     - name: run-virt-customize
-      image: "quay.io/kubevirt/tekton-task-disk-virt-customize:v0.14.0"
+      image: "quay.io/kubevirt/tekton-task-disk-virt:v0.14.0"
       command:
         - entrypoint
       args:
         - '--verbose'
         - $(params.verbose)
       env:
+        - name: COMMAND
```

Comment:
Do we really need the `COMMAND` env or could the command to be executed be the first arg in a step?

Reply:
I've tried both approaches, but I like this one better because I don't know if the arguments are guaranteed in the order. Through the env variable, it is always certain that the command will be there.